src/Middleware.php

```diff
@@ -3,31 +3,138 @@
 namespace Inertia;
 
 use Closure;
-use Illuminate\Support\Facades\Response;
-use Symfony\Component\HttpFoundation\RedirectResponse as Redirect;
+use Illuminate\Http\Request;
+use Illuminate\Support\Collection;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Response;
 
 class Middleware
 {
-    public function handle($request, Closure $next)
+    /**
+     * Determines the current asset version.
+     *
+     * @see https://inertiajs.com/asset-versioning
+     * @param  \Illuminate\Http\Request  $request
+     * @return string|null
+     */
+    public function version(Request $request)
     {
-        $response = $next($request);
+        // When we are running on Laravel Vapor, asset URLs are automatically updated
+        // when the application is being deployed. Because of this, we do not need
+        // to use any files for hashing, as we can simply use this URL instead.
```

Comment:
I don't think we actually need this explanation. It's fairly self explanatory. 🤷‍♂️ 
```suggestion
```

Reply:
Not sure, I personally feel like it might seem like an 'odd' check, as the asset URL might seem fairly generic otherwise?

Reply:
I don't think so, the `asset_url` is a "Laravel thing": https://github.com/laravel/laravel/blob/6bfe68365d0ab1301dfae87867814048440d6a78/config/app.php#L57

And, because this could be in non-Vapor situations as well, it seems odd to have a Vapor specific comment here.

I think we're good to go without the explanation. 👍